include/mbedtls/mbedtls_config.h

```diff
@@ -1029,6 +1029,16 @@
  */
 #define MBEDTLS_PK_PARSE_EC_EXTENDED
 
+/**
+ * \def MBEDTLS_PK_PARSE_EC_COMPRESSED
+ *
+ * Enable the support for parsing public keys of type Short Weierstrass
+ * (MBEDTLS_ECP_DP_SECP_XXX and MBEDTLS_ECP_DP_BP_XXX) which are using the
+ * compressed point format.
+ * Please see MBEDTLS_ECP_PF_COMPRESSED in ecp.h for limitations details.
+ */
+#define MBEDTLS_PK_PARSE_EC_COMPRESSED
```

Comment:
Note: in 3.4 this was not a separate option, support was automatically present in PK when `ECP_C` was defined. So, I think in order to preserve backward compatibility for people using a custom config we need to also auto-enable `MBEDTLS_PK_PARSE_EC_COMPRESSED` when `PK_PARSE_C && ECP_C` in `build_info.h`.